main.go

```diff
@@ -45,9 +46,30 @@ func getTodos(w http.ResponseWriter, r *http.Request) {
 	mu.Lock()
 	defer mu.Unlock()
 
-	w.Header().Set("Content-Type", "application/json")
-	if err := json.NewEncoder(w).Encode(todos); err != nil {
-		http.Error(w, "Failed to encode todos", http.StatusInternalServerError)
+	tmpl := `<!DOCTYPE html>
+<html>
+<head>
+	<title>Todos</title>
```

Comment:
Clearly, this should say:
```suggestion
	<title>Todos!</title>
```